test/unit/hooks/highPingKick/index.test.js

```diff
@@ -0,0 +1,96 @@
+const {HighPingCount} = require('../../../../api/hooks/highPingKick/index.js');
+const { expect } = require('chai');
+
+describe('highPingCount', function () {
+  it('Doesnt do anything when steamid is not found', async () => {
+    sandbox.stub(sails.helpers.sdtdApi, 'getOnlinePlayers').returns(Promise.resolve(
+      [
+        {'steamid':'123456','entityid':97356,'ip':'1.1.1.1','name':'Fake Name','online':true,'position':{'x':81,'y':67,'z':-1030},'experience':-1,'level':3.63118052482605,'health':72,'stamina':79.4040069580078,'zombiekills':7,'playerkills':0,'playerdeaths':1,'score':7,'totalplaytime':0,'lastonline':'0001-01-01T00:00:00','ping':524}
+      ]
+    ));
+    sails.testServerConfig.pingKickEnabled = 1;
+    sandbox.stub(SdtdConfig, 'findOne').returns(Promise.resolve(sails.testServerConfig));
+
+    const highPingCount = new HighPingCount({ });
+
+    await highPingCount.handlePingCheck({
+      server: sails.testServer
+    });
+  });
+  it('Ignores user in whitelist', async () => {
+    sandbox.stub(sails.helpers.sdtdApi, 'getOnlinePlayers').returns(Promise.resolve(
+      [
+        {'steamid':sails.testPlayer.steamId,'entityid':97356,'ip':'1.1.1.1','name':'Fake Name','online':true,'position':{'x':81,'y':67,'z':-1030},'experience':-1,'level':3.63118052482605,'health':72,'stamina':79.4040069580078,'zombiekills':7,'playerkills':0,'playerdeaths':1,'score':7,'totalplaytime':0,'lastonline':'0001-01-01T00:00:00','ping':524}
+      ]
+    ));
+
+    sails.testServerConfig.pingKickEnabled = 1;
+    sails.testServerConfig.pingWhitelist = JSON.stringify([ sails.testPlayer.steamId ]);
+
+    sandbox.stub(SdtdConfig, 'findOne').returns(Promise.resolve(sails.testServerConfig));
+
+    const highPingCount = new HighPingCount({ });
+
+    await highPingCount.handlePingCheck({
```

Comment:
Not `expect`'ing anything because it this would do something, the non stubbed functions would throw, right?

Reply:
Yea I don't know. I could check the redis is set right or maybe kick isn't called? I started this test late last night

Reply:
> kick isn't called
Is probably the best solution I think.

Assuming it will throw works but not so clean IMO